tests/acceptance/conftest.py

```diff
@@ -75,33 +73,14 @@ def app(request):
 
 @pytest.fixture
 def arsenic(selenium, app):
-    return Arsenic(selenium)
-
-
-@pytest.fixture(scope='session', autouse=True)
-def cleanup_cookies():
-    cookies_path = './tests/acceptance/login_cookies.pkl'
-    if os.path.exists(cookies_path):
-        os.unlink(cookies_path)
+    Arsenic(selenium)
 
 
 @pytest.fixture
 def login(arsenic):
-    cookies_path = './tests/acceptance/login_cookies.pkl'
-    if not os.path.exists(cookies_path):
-        top_navigation_page.log_in('admin@inspirehep.net', '123456')
-        sleep(1)
-        cookies = arsenic.get_cookies()
-        with open(cookies_path, 'w') as cookies_fd:
-            pickle.dump(cookies, cookies_fd)
-    else:
-        with open(cookies_path) as cookies_fd:
-            cookies = pickle.load(cookies_fd)
-        for cookie in cookies:
-            # selenium driver only allows setting the current domain by
-            # accessing a url from it, we get one that loads fast
-            top_navigation_page.ping()
-            arsenic.add_cookie(cookie)
+    top_navigation_page.log_in('admin@inspirehep.net', '123456')
```

Comment:
Answering to myself, that was because the previous idea was that instead of login and logout on every tests, we just wanted to login once per session, to keep it faster.